app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -106,6 +107,7 @@ class MainActivity : AppCompatActivity(), ReplayModeFragment.OnSelectModeItemLis
     private var modelPerformance = ModelPerformance.On(ModelPerformanceMode.FIXED, ModelPerformanceRate.HIGH)
     private var lastSpeed = 0f
     private var calibrationProgress = 0f
+    private var playbackTracked = false
```

Comment:
IMO not very clear name, maybe smth like `isProgressChanging` ?

Reply:
let's try this, I've changed this name 3 times
I propose isProgressChangingByUser

Reply:
I'd stick to `isProgressChanging`, I do not see situation when non-user could change progress. Even if it could happen I believe logic will remain the same.

Reply:
It could be changed by some module\service\code etc.
I'll change to isProgressChanging as it is clear for you both outside the detailed context